stories/core/DropDownMenu.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { DropDownMenu } from 'react-ui-core/src'
+
+export const DefaultDropDownMenu = (
+  <DropDownMenu
+    options={['Option1', 'Option2', 'Option3']}
+  />
+)
+
+export const DropDownMenuOnSelection = (
+  <DropDownMenu
+    options={['Option1', 'Option2', 'Option3']}
+    onSelection={value => {
+      alert(value)
```

Comment:
Can you use the actions addon instead here?  Take a look at the Modal story for an example.  https://github.com/rentpath/react-ui/blob/master/stories/core/Modal.js#L25